config/development.toml

```diff
@@ -179,33 +179,33 @@ ideal = { country = "NL", currency = "EUR" }
 
 [pm_filters.braintree]
 paypal = { currency = "AUD,BRL,CAD,CNY,CZK,DKK,EUR,HKD,HUF,ILS,JPY,MYR,MXN,TWD,NZD,NOK,PHP,PLN,GBP,RUB,SGD,SEK,CHF,THB,USD" }
-credit = { not_available_flows = {capture_method="manual"} }
-debit =  { not_available_flows = {capture_method="manual"} }
+credit = { not_available_flows = { capture_method = "manual" } }
```

Comment:
are these formatting intentional?

Reply:
Cargo fmt, did this formatting, not done intentionally